universal-profiling-integration/src/main/java/co/elastic/otel/UniversalProfilingProcessor.java

```diff
@@ -99,9 +105,18 @@ public static UniversalProfilingProcessorBuilder builder(SpanProcessor next, Res
                 + " There must be at most one processor of this type active at a time!");
       }
 
+      long initialSpanDelay;
+      if (activeOnlyAfterProfilerRegistration) {
+        initialSpanDelay = 0; // do not buffer spans until we know that a profiler is running
+        tlsPropagationActive = false;
+      } else {
+        initialSpanDelay = INITIAL_SPAN_DELAY_NANOS; // delay conservatively to not miss any data
```

Comment:
[question] from this I understand that "span delay" here means the time duration for which the spans are buffered, is that correct ? If so, then probably renaming this to "span buffering duration" or similar seems more appropriate.

Reply:
Correct, I'll rename it.